Sources/MotorControl/Views/TappingStepView.swift

```diff
@@ -65,8 +65,9 @@ struct TappingStepView: View {
     }
     
     @ViewBuilder
-    private func singleTappingButton(target: TappingButtonIdentifier) -> some View {
+    private func singleTappingButton(target: TappingButtonIdentifier, buttonId: String) -> some View {
         Text("Tap", bundle: SharedResources.bundle)
+            .accessibilityLabel(buttonId)
```

Comment:
So... why not just use `target.rawValue` or `"tap_\(target.rawValue)"` or something like that?